### PR TITLE
Make consent cookie value public

### LINKED ISSUE #1590: Add API to the ITrackingConsentFeature to generate the cookie for javascript use
Damian wants to set the consent cookie from a javascript action, but wants to honor all of the middleware settings for it. Add an API to the feature interface that generates the full cookie for you.

### PR DESCRIPTION
**Pull request by user1336**
Fixes #1590

### CODE REVIEW

File: src/Microsoft.AspNetCore.CookiePolicy/ResponseCookiesWrapper.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace Microsoft.AspNetCore.CookiePolicy
 {
     internal class ResponseCookiesWrapper : IResponseCookies, ITrackingConsentFeature
```
Comment (HaoK) [inline src/Microsoft.AspNetCore.CookiePolicy/ResponseCookiesWrapper.cs]: The class is internal so this won't help much

Comment (user1336) [reply src/Microsoft.AspNetCore.CookiePolicy/ResponseCookiesWrapper.cs]: How did I miss that, should I add a Defaults class?

Comment (Tratcher) [reply src/Microsoft.AspNetCore.CookiePolicy/ResponseCookiesWrapper.cs]: Wait, Damian gave me some additional feedback yesterday that may make this irrelevant.

**Comment by Tratcher** [comment]
Closing this for now as the requirements have significantly changed.

**Pull request closed without merging**